### PR TITLE
Added Node Buffer to auth0 cdn example

### PR DESCRIPTION
**Pull request by mbailey-arkoselabs**

### CODE REVIEW

**Review by jasonramke — APPROVED** [review]
This will also need to be added to the [cdn-integration](https://github.com/ArkoseLabs/cdn-integration) repo, because this repo is public we usually make a change there first and then copy it to this repo once approved.

**Review by davidtsuji — APPROVED** [review]
approving as a code owner

**Pull request merged**
